engine/movie/intro_yellow.asm

```diff
@@ -708,14 +708,14 @@ YellowIntro_CheckFrameTimerDecrement:
 	ret
 
 .asm_f9e4b
-	vc_hook FPA_031801_End2
+	vc_hook Stop_Reducing_intro_scene_flashing_0f
 	scf
 	ret
 
 YellowIntro_LoadDMGPalAndIncrementCounter:
 	ld hl, wYellowIntroSceneTimer
 	ld a, [hl]
-	vc_hook FPA_031801_End1
+	vc_hook Stop_Reducing_intro_scene_flashing_pika_thunderbolt
```

Comment:
How about just use the raw index values for now, like you did with 0F, and we can update them whenever we add constants for the intro scenes?
```suggestion
	vc_hook Stop_reducing_intro_scene_flashing_??
```